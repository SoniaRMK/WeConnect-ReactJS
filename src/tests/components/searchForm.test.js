import {shallow} from 'enzyme';
import React from 'react';
import SearchFilters from '../../components/business/searchForm';

describe('SearchForm component',() => {

    it('should match snapshot',() =>{
        const wrapper = shallow(<SearchFilters/>)
        expect(wrapper).toMatchSnapshot();
    })
})
