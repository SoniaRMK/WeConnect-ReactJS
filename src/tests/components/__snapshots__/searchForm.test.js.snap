// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`SearchForm component should match snapshot 1`] = `
<form
  className="form-inline"
>
  <Categories />
  <Locations />
  <input
    className="form-control mb-2 mr-sm-2"
    name="search_term"
    placeholder="Enter search term"
    type="text"
  />
  <button
    className="btn btn-info mb-2"
    type="submit"
  >
    Search
  </button>
   Â 
</form>
`;
